proofs/src/plonk/keygen.rs

```diff
@@ -15,14 +15,15 @@ use super::{
 };
 use crate::{
     circuit::Value,
-    dev::cost_model::cost_model_options,
     poly::{
         batch_invert_rational,
         commitment::{Params, PolynomialCommitmentScheme},
         EvaluationDomain, ExtendedLagrangeCoeff,
     },
     utils::{arithmetic::parallelize, rational::Rational},
 };
+#[cfg(feature = "cost-estimator")]
```

Comment:
I believe we are removing (have removed) this feature.

Reply:
Removed in the rebase